tests/flow/test_graph_merge.py

```diff
@@ -675,3 +675,13 @@ def test31_alias_multiple_definition(self):
         except redis.exceptions.ResponseError as e:
             # Expecting an error.
             assert("can't be redeclared in a MERGE clause" in str(e))
+
+    def test32_reset_op(self):
+        redis_con = self.env.getConnection()
+        graph = Graph(redis_con, "reset_op")
+
+        res = graph.query("CREATE (a:A), (b:B)")
+        self.env.assertEquals(res.nodes_created, 2)
+
+        res = graph.query("MATCH (a:A), (b:B) SET a:X MERGE (c:C) MERGE (d:D)")
+        self.env.assertEquals(res.nodes_created, 2)
```

Comment:
Please add a comment explaining the reasoning behind this test, something along the line of the X_OP doesn't register a `reset` function...